js/localization/messages/tr.json

```diff
@@ -0,0 +1,261 @@
+        "dxDataGrid-sortingAscendingText": "Artan Sıralama",
+        "dxDataGrid-sortingDescendingText": "Azalan Sıralama",
+        "dxDataGrid-sortingClearText": "Sıralamayı Temizle",
+        "dxDataGrid-editingSaveAllChanges": "Değişiklikleri Kaydet",
+        "dxDataGrid-editingCancelAllChanges": "Değişiklikleri iptal et",
+        "dxDataGrid-editingAddRow": "Satır ekle",
+        "dxDataGrid-summaryMin": "Min: {0}",
+        "dxDataGrid-summaryMinOtherColumn": "{1} min: {0}",
+        "dxDataGrid-summaryMax": "Max: {0}",
+        "dxDataGrid-summaryMaxOtherColumn": "{1} max: {0}",
+        "dxDataGrid-summaryAvg": "Ort: {0}",
+        "dxDataGrid-summaryAvgOtherColumn": "{1} ortalaması: {0}",
+        "dxDataGrid-summarySum": "Top: {0}",
+        "dxDataGrid-summarySumOtherColumn": "{1} toplamı: {0}",
+        "dxDataGrid-summaryCount": "Toplam: {0}",
+        "dxDataGrid-columnFixingFix": "Sabitle",
+        "dxDataGrid-columnFixingUnfix": "Çöz",
+        "dxDataGrid-columnFixingLeftPosition": "Sola",
+        "dxDataGrid-columnFixingRightPosition": "Sağa",
+        "dxDataGrid-exportTo": "Dışa aktar",
+        "dxDataGrid-exportToExcel": "Excel dosyasına aktar",
+        "dxDataGrid-excelFormat": "Excel dosyası",
+        "dxDataGrid-selectedRows": "Seçili satırlar",
+        "dxDataGrid-exportSelectedRows": "Seçili satırları aktar",
+        "dxDataGrid-exportAll": "Tüm verileri dışa aktar",
+        "dxDataGrid-headerFilterEmptyValue": "(Blanks)",
+        "dxDataGrid-headerFilterOK": "Tamam",
+        "dxDataGrid-headerFilterCancel": "İptal",
+        "dxDataGrid-ariaColumn": "Sütun",
+        "dxDataGrid-ariaValue": "Veri",
+        "dxDataGrid-ariaFilterCell": "Filtre hücresi",
+        "dxDataGrid-ariaCollapse": "Daralt",
+        "dxDataGrid-ariaExpand": "Genişlet",
+        "dxDataGrid-ariaDataGrid": "Tablo",
+        "dxDataGrid-ariaSearchInGrid": "Tabloda ara",
+        "dxDataGrid-ariaSelectAll": "Hepsini seç",
+        "dxDataGrid-ariaSelectRow": "Satırı seç",
+        "dxDataGrid-filterBuilderPopupTitle": "Filtre Oluşturucu",
+        "dxDataGrid-filterPanelCreateFilter": "Filtre Oluştur",
+        "dxDataGrid-filterPanelClearFilter": "Temizle",
+        "dxDataGrid-filterPanelFilterEnabledHint": "Filtreyi etkinleştir",
+        "dxTreeList-ariaTreeList": "Ağaç listesi",
+        "dxTreeList-editingAddRowToNode": "Ekle",
+        "dxPager-infoText": "Sayfa {0} / {1} ({2} veri)",
+        "dxPager-pagesCountText": "arasında",
+        "dxPivotGrid-grandTotal": "Genel Toplam",
+        "dxPivotGrid-total": "{0} Toplam",
+        "dxPivotGrid-fieldChooserTitle": "Alan Seçici",
+        "dxPivotGrid-showFieldChooser": "Alan Seçiciyi Göster",
+        "dxPivotGrid-expandAll": "Tümünü Genişlet",
+        "dxPivotGrid-collapseAll": "Tümünü Daralt",
+        "dxPivotGrid-sortColumnBySummary": "Sort \"{0}\" by This Column",
+        "dxPivotGrid-sortRowBySummary": "Sort \"{0}\" by This Row",
```

Comment:
The `dxPivotGrid-sortColumnBySummary` and `dxPivotGrid-sortRowBySummary` messages look untranslated. Can you add right messages here please?